File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>
void    left_colums(int width, int height, int j);
void    right_colums(int width, int height, int j);




int main(void)
{
    int height;
    int width;
    int i;
    int j;

    i = 0;
    do  //Loop to get Height, will Do until a number between 1 and 8, inclusive, is enter
    {
        height = get_int("Height: ");
    }
    while (height < 1  || height > 8);
    width = height;
    while (i < height)
    {
        j = width;
        left_colums(width, height, j);
        printf("  ");
        j = 0;
        right_colums(width, height, j);
        printf("\n");
        height--;
    }
}
void    left_colums(int width, int height, int j)
{
    while (j > (width - height + 1))// Checks number of empty spaces needed
    {
        printf(" ");
        j--;
    }
    while (j <= (width - height + 1) && j != 0) //prints # needed
    {
        printf("#");
        j--;
    }
}

void    right_colums(int width, int height, int j)
{
    while (j <= (width - height))  //prints # needed, no need for empty spaces, outside function will print \n
    {
        printf("#");
        j++;
    }
}

File: sentimental-mario-more/mario.py
import cs50
# TODO


def main():
    while True:
        height = cs50.get_int("Height: ")
        if height > 0 and height <= 8:
            break
    width = height

    i = 0
    while (i < height):
        j = width
        left_colums(width, height, j)
        print("  ", end="")
        j = 0
        right_colums(width, height, j)
        print("\n", end="")
        height -= 1


def left_colums(width, height, j):
    while (j > (width - height + 1)):
        print(" ", end="")
        j -= 1
    while (j <= (width - height + 1) and j != 0):
        print("#", end="")
        j -= 1


def right_colums(width, height, j):
    while (j <= (width - height)):
        print("#", end="")
        j += 1


main()